boot/storage.txt

```diff
@@ -0,0 +1,485 @@
+
+    +--------------+------------------------+----------------------------------+
+    |  Filesystem  |  Package               |  Command                         |
+    +--------------+------------------------+----------------------------------+
+    |              |                        |                                  |
+    |  swap        | core/busybox           |  mkswap                          |
+    |  EXT2/3/4    | extra/e2fsprogs        |  mkfs.ext2/3/4                   |
+    |  DOS/FATxx   | extra/dosfstools       |  mkfs.dos/fat/vat                |
+    |  XFS         | extra/xfsprogs         |  mkfs.xfs                        |
+    |  BTRFS       | community/btrfs-progs  |  mkfs.btrfs                      |
+    |  NTFS        | community/ntfs-3g      |  mkfs.ntfs                       |
+    |              |                        |                                  |
+    +--------------+------------------------+----------------------------------+
+    
+    Other filesystems exist with varying degrees of popularity, including JFS,
+    ReiserFS, and ZFS. The community repository is an excellent place to share
+    work in including these filesystems in KISS! Others are available in 
+    user-created repositories. ZFS is one such example [8]. Due to licensing 
+    restrictions, ZFS requires additional work to use than other filesystems.
+
+    EXT4 is a solid, general purpose filesystem. For UEFI systems, FAT32 is a 
+    required filesystem for the BOOT partition. BTRFS is an experimental 
+    filesystem which sees heavily development by organizations like Facebook,
+    and offers a different way of managing storage than more traditional (EXT,
+    DOS, NTFS) filesystems. All of these have built-in kernel support.
+
+    For a more complete list of other filesystem options and what limitations 
+    and features exist for them, refer to the Arch Wiki [9].
+
+
+    [3.2] Swap Partition
+    ____________________________________________________________________________ 
+    
+    To set up a partition as a Linux swap space, the mkswap command is use. 
+    Replace X with the drive where the swap partition is located, and Y by the
+    partition on that drive that will be formatted as swap.
+    
+    +--------------------------------------------------------------------------+
+    |                                                                          |
+    |   mkswap /dev/sdXY                                                       |
+    |   swapon /dev/sdXY                                                       |
+    |                                                                          |
+    +--------------------------------------------------------------------------+  
+
+
+    [3.3] Swap file
+    ____________________________________________________________________________    
+    As an alternative to creating an entire partition, a swap file offers 
```

Comment:
Add blank line above this one.

Reply:
Do you mean there's an extra blank at 345?

Reply:
```
    ____________________________________________________________________________    
    As an alternative to creating an entire partition, a swap file offers 
```

```
    ____________________________________________________________________________    

    As an alternative to creating an entire partition, a swap file offers 
```

Reply:
I see it now! It seems that the dividing line was using hyphens instead of underscores? I've resolved this.

How are your eyes so keen.